### PR TITLE
BeaconEquipment added and example VehicleType modeling for EPIAP

### PR DESCRIPTION
**Pull request by ue71603**
The new BeaconEquipment is added after a discussion with @haeckerbaer.  I updated the example of @klement-MENTZ and added it as well. 
Please check. I think we will have to update it once or twice.

### CODE REVIEW

**Comment by Aurige** [comment]
That's Ok for me but, since we already have a BeaconPoint, I would add a BeaconPointRef in this new Equipment and/or a Ref to this (these ?)  Equipment in the BEaconPoint.

**Review by Aurige — APPROVED** [review]

**Comment by Aurige** [comment]
Ooops, I approved it after our discussion ... but didn't see that Travis raised an issue

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 1]
```diff
@@ -134,6 +134,95 @@ Rail transport, Roads and Road transport
...
 	<!-- ======================================================================= -->
+	<xsd:simpleType name="PassengerBeaconEquipmentIdType">
+		<xsd:annotation>
+			<xsd:documentation>Type for identifier of a BEACON EQUIPMENT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="PassengerEquipmentIdType"/>
+	</xsd:simpleType>
+	<xsd:element name="PassengerBeaconEquipmentRef" type="PassengerBeaconEquipmentRefStructure" substitutionGroup="PassengerEquipmentRef">
+		<xsd:annotation>
+			<xsd:documentation>Reference of an BEACON  EQUIPMENT.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:145]:
```suggestion
			<xsd:documentation>Reference of an BEACON EQUIPMENT.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 2]
```diff
@@ -134,6 +134,95 @@ Rail transport, Roads and Road transport
...
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="PassengerBeaconEquipmentRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a reference to an BEACON  EQUIPMENT.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:150]:
```suggestion
			<xsd:documentation>Type for a reference to an BEACON EQUIPMENT.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 3]
```diff
@@ -134,6 +134,95 @@ Rail transport, Roads and Road transport
...
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="ble">
+				<xsd:annotation>
+					<xsd:documentation>bluetooth low energy</xsd:documentation>
+				</xsd:annotation>			
+			</xsd:enumeration>
+			<xsd:enumeration value="uwb">
+				<xsd:annotation>
+					<xsd:documentation>ultra wiede band</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:174]:
```suggestion
					<xsd:documentation>ultra wide band</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 4]
```diff
@@ -134,6 +134,95 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>			
+			</xsd:enumeration>
+			<xsd:enumeration value="wifi">
+				<xsd:annotation>
+					<xsd:documentation>In some cases the beacon is a wifi hot spot.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:179]: Should this be something like "BSSID"?

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 5]
```diff
@@ -134,6 +134,95 @@ Rail transport, Roads and Road transport
...
+			<xsd:restriction base="PassengerEquipmentRefStructure">
+				<xsd:attribute name="ref" type="PassengerBeaconEquipmentIdType" use="required">
+					<xsd:annotation>
+						<xsd:documentation>Identifier of a BEACON EQUIPMENT.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:attribute>
+			</xsd:restriction>
+		</xsd:simpleContent>				
+	</xsd:complexType>	
+		<xsd:simpleType name="BeaconTypeEnumeration">
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:162]:
```suggestion
	<xsd:simpleType name="BeaconTypeEnumeration">
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 6]
```diff
@@ -134,6 +134,95 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>			
+			</xsd:enumeration>
+			<xsd:enumeration value="bidirectional">
+				<xsd:annotation>
+					<xsd:documentation>The beacon is able to do bidirectional transmissions.</xsd:documentation>
+				</xsd:annotation>			
+			</xsd:enumeration>
+		</xsd:restriction>
+	</xsd:simpleType>	
+
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:202]:
```suggestion

```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 7]
```diff
@@ -134,6 +134,95 @@ Rail transport, Roads and Road transport
...
+		<xsd:simpleType name="BeaconProtocolEnumeration">
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:203]:
```suggestion
	<xsd:simpleType name="BeaconProtocolEnumeration">
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 8]
```diff
@@ -134,6 +134,94 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>			
+			</xsd:enumeration>
+			<xsd:enumeration value="bidirectional">
+				<xsd:annotation>
+					<xsd:documentation>The beacon is able to do bidirectional transmissions.</xsd:documentation>
+				</xsd:annotation>			
+			</xsd:enumeration>
+		</xsd:restriction>
+	</xsd:simpleType>	
+		<xsd:simpleType name="BeaconProtocolEnumeration">
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:202]:
```suggestion
	<xsd:simpleType name="BeaconProtocolEnumeration">
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd (hunk 4 above)
Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:179]: @skinkie :

**Comment by nick-knowles** [comment]
NeTEx UML Diagram added.

**Comment by nick-knowles** [comment]
[](url)
image

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 9]
```diff
@@ -116,7 +116,7 @@ Rail transport, Roads and Road transport
 	</xsd:simpleType>
 	<xsd:element name="HelpPointEquipmentRef" type="HelpPointEquipmentRefStructure" substitutionGroup="PassengerEquipmentRef">
 		<xsd:annotation>
-			<xsd:documentation>Identifier of an HELP POINT EQUIPMENT.</xsd:documentation>
+			<xsd:documentation>Reference  of an HELP POINT EQUIPMENT.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:119]:
```suggestion
			<xsd:documentation>Reference of an HELP POINT EQUIPMENT.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 10]
```diff
@@ -134,6 +134,94 @@ Rail transport, Roads and Road transport
...
+	</xsd:simpleType>
+
+		<xsd:simpleType name="BeaconDirectionEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for communication direction of beacon.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="unidirectional">
+				<xsd:annotation>
+					<xsd:documentation>normal unidirectional beacon. Usually a backend is then mandatory.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:192]:
```suggestion
					<xsd:documentation>Normal unidirectional beacon. Usually a backend is then mandatory.</xsd:documentation>
```

**Review by skinkie — APPROVED** [review]

**Pull request merged**